freestyle/shared/src/test/scala/freestyle/free.scala

```diff
@@ -129,6 +129,25 @@ class freeTests extends WordSpec with Matchers {
       "implicitly[SCtors1[SCtors1.Op]]" should compile 
     }
 
+    "allow implicit parameters in algebra members" in {
```

Comment:
ok, but you've caught me literally as I was about to go to bed so maybe somebody could add these tomorrow? :smile: